StackTask.playground/Contents.swift

```diff
@@ -0,0 +1,96 @@
+// Protocol contains methods for working with Stack class
+protocol StackOperations {
+    func push (element: Int)
+    func pop() -> Int?
+    func top() -> Int?
+}
+
+class Stack: StackOperations {
+    fileprivate var stackArray: [Int] = []
+
+    // Adds element on the top of the stack
+    func push(element: Int) {
+        stackArray.append(element)
+    }
+    
+    // Function removes and returns the top element of the stack. returns nil when the invoking stack is empty.
+    func pop() -> Int? {
+        if stackArray.isEmpty {
+            return nil
+        } else {
+            let lastStackItem = stackArray[stackArray.count-1]
+            stackArray.remove(at: stackArray.count-1)
+            return lastStackItem
+        }
+    }
+    
+    // Function returns the top element of the stack, but does not remove it.
+    func top() -> Int? {
+        stackArray.isEmpty ? nil : stackArray[stackArray.count-1]
+    }
+    
+    // Function for output every items from stack (from last added to first)
+    func printStack() {
+        let elementIndex = stackArray.count - 1
+        if stackArray.isEmpty {
+            print("Stack is empty")
+        }
+        for i in 0..<stackArray.count {
+            print(stackArray[elementIndex - i], terminator: " ")
+        }
+        print()
+    }
+}
+
+class StackStatistic: Stack {
```

Comment:
```suggestion
class StackStatistics: Stack {
```

Please read the task one more time.

Reply:
ok)